var/spack/repos/builtin/packages/py-melissa-core/package.py

```diff
@@ -18,6 +18,8 @@ class PyMelissaCore(PythonPackage):
     maintainers("robcaulk", "mschouler", "raffino")
 
     version("develop", branch="develop")
```

Comment:
Should we set this version to be preferred? You can check which version is used by default with `spack info py-melissa-core`.

Reply:
Good point, I have set `develop` to preferred.